.spin/bootstrap

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Running the scripts here helps Spin to do it in background
+# and then make a snapshot to save time at new instance creation.
+
+./bazel version
+
+# TODO: Make sure you have "/home/spin/.bazel_binaries/4.1.0/bin_t" in spin user path too.
+export PATH="/home/spin/.bazel_binaries/4.1.0/bin_t:$PATH"
+# Sets up go to definition in VS Code
+tools/scripts/build_compilation_db.sh
+
+
+# TODO: useful aliases
+
+# alias compile_sorbet="./bazel build //main:sorbet --config=dbg"
+
+# comp_run() {
+# compile_sorbet && sorbet_run $@
+# }
+
+# sorbet_run() {
+#   bazel-bin/main/sorbet $@ --silence-dev-message
+# }
+
+# bazel_test_find() {
+#   ./bazel query ‘tests(//...)’ | grep $@ | xargs ./bazel test --config=dbg --test_output=errors
+# }
```

Comment:
Is it possible to append these automatically to the user's zshrc as a last step without messing up the user's own dotfiles config?

Reply:
Merged! Can start hacking 🛠